### DISCUSSION TITLE
Are the azure certifications mandatory to proceed?

### DISCUSSION THREAD

**Discussion by manavmodi22** [github/india#252] [Students]
In the self-learning section, it shows the certification upload link. Is it mandatory to get certified through paid exam after the MS learn pathway?

**Comment by balasai45** [comment]
same question

**Comment by Mik-27** [comment]
I guess it is for those individuals who have already completed the certification so they don't have to go through the learning again and their progress will automatically be updated.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
